### PR TITLE
✨ [DRAFT: RAW]: Security policy support

### PR DESCRIPTION
**Pull request by laurentsimon**
PR separates checks from policy for the Security-Policy check.
No breaking changes on output.

### CODE REVIEW

File: checks/raw/binary_artifact.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ import (
 
 // BinaryArtifacts retrieves the raw data for the Binary-Artifacts check.
 func BinaryArtifacts(c clients.RepoClient) (checker.BinaryArtifactData, error) {
-	var files []checker.File
+	files := []checker.File{}
```
Comment (azeemshaikh38) [inline checks/raw/binary_artifact.go:33]: If you want to create an empty slice prefer: `make([]checker.File, 0)` to make the intent explicit.

File: docs/checks/internal/validate/main.go [hunk 2]
```diff
@@ -70,6 +71,10 @@ func listCheckFiles() (map[string]string, error) {
 			continue
 		}
 
+		// WARNING: assume the filename is the same during migration
```
Comment (azeemshaikh38) [inline docs/checks/internal/validate/main.go:74]: Unrelated to this PR - can we do away with these validations where we string parse the checks and the API usage?

**Review by azeemshaikh38 — APPROVED** [review]
Thanks!

File: checks/raw/binary_artifact.go (hunk 1 above)
Comment (laurentsimon) [reply checks/raw/binary_artifact.go:33]: thanks.

File: docs/checks/internal/validate/main.go (hunk 2 above)
Comment (laurentsimon) [reply docs/checks/internal/validate/main.go:74]: Using a CFG is better but requires more work. If you have something specific in mind that's easy to implement, sure. The string based approach currently works, so I don't see this as a priority unless it keeps failing.
What do you have in mind?

**Pull request merged**